x-pack/platform/plugins/shared/stack_connectors/public/connector_types/inference/translations.ts

```diff
@@ -105,3 +105,10 @@ export const COPIED_TOOLTIP = i18n.translate(
     defaultMessage: 'Copied!',
   }
 );
+
+export const BODY_INVALID = i18n.translate(
+  'xpack.stackConnectors.components.inference.error.invalidBodyText',
+  {
+    defaultMessage: 'Body does not have a valid JSON format.',
```

Comment:
In my head the wording here can be improved, just don't know how exactly :) Perhaps the docs/writing team can take a peek here?